src/resources/parser/glb-parser.js

```diff
@@ -958,28 +955,21 @@ const createMaterial = function (gltfMaterial, textures, disableFlipV) {
             }
         }
 
-        let scale = uvONE;
-        let offset = uvZERO;
-
-        const extensions = source.extensions;
-        if (extensions) {
-            const textureTransformData = extensions.KHR_texture_transform;
-            if (textureTransformData) {
-                if (textureTransformData.scale) {
-                    scale = textureTransformData.scale;
-                }
-                if (textureTransformData.offset) {
-                    offset = textureTransformData.offset;
-                }
+        const textureTransform = source?.extensions?.KHR_texture_transform;
```

Comment:
You don't need the first ? because `source` is valid.

Reply:
thanks!